app/assets/stylesheets/header.scss

```diff
@@ -5,6 +5,25 @@
   width: 67px;
 }
 
+.navbar {
+  background-color: $gray-image-color;
+  font-family: 'Arvo', sans-serif;
+  font-size: 135%;
+  display: inline-block;
+  min-height: 80px;
+  width: 100vw;
+}
+
+@media (min-width: 768px) {
+  .navbar .navbar-nav {
+    text-align: center;
+    justify-content: center;
+    float: none;
+    padding-bottom: 1vh;
+    width: 100vw;
+  }
+}
+
 // Dropdown for adding a new product or vendor
 .dropdown-content form:hover {
   background-color: $gray-color;
```

Comment:
The color of me hovering over an item in the dropdown is almost black. You can barely make out the text.

Reply:
You're on the wrong branch -- try switching to `discover-ui`; I think you're on the old corrupted reference branch `discoverY-ui`